flyerhq/flutter_chat_ui
# feature to tag users in group chat

### What does it do?

was looking for a way to popup a list of users to tag when you type '@' like WhatsApp. this PR implements optional parameter `mentions` which accepts list of `Mention` objects to add mentions data

### Why is it needed?

Needed a feature to tag users in a group chat

### How to test it?

add `mentions` parameter in `Chat` widget add `Mention(trigger: '@', data: [{'id': '1', 'display': 'John Doe'}])` to mentions, type `@` in input and you'll see list of users provided in data in mention

### Related issues/PRs
Not related to any issues/PRs
